pkg/controllers/jobset_controller.go

```diff
@@ -255,10 +255,13 @@ func (r *JobSetReconciler) resumeJobSetIfNecessary(ctx context.Context, js *jobs
 				log.Error(nil, "job missing ReplicatedJobName label")
 			}
 			job.Spec.Suspend = pointer.Bool(false)
-			job.Status.StartTime = nil
 			if err := r.Update(ctx, job); err != nil {
 				return err
 			}
+			job.Status.StartTime = nil
```

Comment:
please check if StartTime is already nil and avoid the update if it is

Reply:
In reconcile we call this function only if the job is suspended. But ok.